optuna/cli.py

```diff
@@ -126,9 +124,7 @@ def get_parser(self, prog_name):
             "--study", default=None, help="This argument is deprecated. Use --study-name instead.",
         )
         parser.add_argument(
-            "--study-name",
-            default=None,
-            help="A human-readable name of a study to distinguish it from others.",
+            "--study-name", default=None, help="The name of the study to set attribute to.",
```

Comment:
I’m not fully sure, but `attribute` seems to be countable. Do we need an article?
```suggestion
            "--study-name", default=None, help="The name of the study to set the attribute to.",
```

Reply:
Done in 00dfd8f